src/components/custom/_agreement-label.js

```diff
@@ -51,13 +44,17 @@ const AgreementLabel = ({ handleChangeCheckbox, isChecked, color }) => {
                     fontSize="var(--text-size-xs)"
                     translate_text="I agree to the <0>terms and conditions</0>"
                     components={[
-                        <LinkText
+                        <LocalizedLinkText
                             key={0}
-                            href={url}
+                            type="terms-and-conditions"
+                            external="true"
                             target="_blank"
```

Comment:
```suggestion
```
Since using type `terms-and-conditions` should automatically set it to open in a new tab (based on the comment in the localized-link). This shouldn't be necessary

Reply:
i don't know why but it doesn't open new window without target="_blank". And componemt-guide.md of the project says to use external link with target=' _blank'

Reply:
done